pkg/common/cns-lib/volume/listview.go

```diff
@@ -0,0 +1,280 @@
+package volume
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/vmware/govmomi/property"
+	"github.com/vmware/govmomi/view"
+	"github.com/vmware/govmomi/vim25/types"
+
+	cnsvsphere "sigs.k8s.io/vsphere-csi-driver/v2/pkg/common/cns-lib/vsphere"
+	"sigs.k8s.io/vsphere-csi-driver/v2/pkg/csi/service/logger"
+)
+
+const (
+	waitForUpdatesRetry = 1 * time.Minute
+	infoPropertyName    = "info"
+)
+
+// ListViewImpl is the struct used to manage a single listView instance.
+// params
+// virtualCenter: holds a reference to the global VC object
```

Comment:
Can we place comments in the order of variables listed in the struct

Reply:
Fixed.